src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString.trim().split(';');
+  const styleObject = {};
+
+  for (const item of styleDeclarations) {
+    if (item.trim().length > 0) {
+      const resultItemSplit = item.split(':');
```

Comment:
Use forEach method instead for of loop
```suggestion
 styleDeclarations.forEach(() => {
      const resultItemSplit = item.split(':');
```